src/main/java/org/apache/maven/reporting/MavenReport.java

```diff
@@ -53,7 +53,9 @@ public interface MavenReport {
     void generate(Sink sink, Locale locale) throws MavenReportException;
 
     /**
-     * Get the base name used to create report's output file(s).
+     * Get the output name denoting a base location relative to the {@link #getReportOutputDirectory()}
```

Comment:
This one is a little hard to describe because what the API does is complicated. On first read, I thought this was just a file name but it's not. But now I read it a third time and maybe it is just a file name? I don't know.

If it is just a file name, then "Returns the name of the output file that will be written in {@link #getReportOutputDirectory()}"

If it can contain subdirectories, then "Returns a path relative to {@link #getReportOutputDirectory()} where the output file will be written."

Reply:
@elharo,what it contains (file or subdirectory) depends on whether it is a single or multi page report. Javadoc would be an example for multi page.

Reply:
OK, so maybe it's a directory name? All of this should be laid out in the docs. What might help is a longish class comment explaining where the report files go and how they're configured.

Reply:
@elharo, it is both. A path to a file. I need to rename the API to satisfy this. I could do with a default interface method. The current name is just bad. I am happy to accept any reformulation of the docs which make it better. Should I raise a PR to rename the method and deprecate it?